tests/proxy/handler.js

```diff
@@ -27,7 +27,14 @@ module.exports.checkRequest = ( request, h ) => {
 					body.query
 				);
 			}
-			return require( './mocks/ads/recommendations/results.json' );
+
+			if (
+				body.query.includes( 'IMPROVE_PERFORMANCE_MAX_AD_STRENGTH' )
+			) {
+				return require( './mocks/ads/recommendations/pmax-asset.json' );
+			}
+
+			return require( './mocks/ads/recommendations/campaign-budget.json' );
```

Comment:
Not a blocker

Do we have mock data for combination of different types it help us to get clear idea about the shape?
- `CAMPAIGN_BUDGET, MARGINAL_ROI_CAMPAIGN_BUDGET`
- `IMPROVE_PERFORMANCE_MAX_AD_STRENGTH, CAMPAIGN_BUDGET`
- `IMPROVE_PERFORMANCE_MAX_AD_STRENGTH, CAMPAIGN_BUDGET, MARGINAL_ROI_CAMPAIGN_BUDGET`